scripts/ssl-enum-ciphers.nse

```diff
@@ -1149,6 +1151,15 @@ action = function(host, port)
     return nil
   end
 
+  -- Store the discovered ciphers in host table, should be utilized by other
+  -- scripts to find cipher suite available.
+  for key, val in pairs(results) do
+    host.registry.tls_protos[key] = {}
```

Comment:
port table is not persistent: may not be the same table passed to multiple scripts. The host registry is the right place to store this, but it should be keyed on the port number and protocol to make sure we don't mix them up.